tlslite/utils/codec.py

```diff
@@ -45,17 +209,24 @@ def addVarTupleSeq(self, seq, length, lengthLength):
         @type lengthLength: int
         @param lengthLength: length in bytes of overall length field
         """
-        if len(seq) == 0:
+        if not seq:
```

Comment:
actually checking for boolean on an array/dictionary/string to see if it is empty is considered more pythonic

all collection types are True when not empty and False when empty, correctly handling None is a nice side effect

it also may also be faster, but I haven't tested it